pkg/credentialstores/secret.go

```diff
@@ -0,0 +1,91 @@
+package credentialstores
+
+import (
```

Comment:
Imports are typically in this order:
 - standard
 - other
 - kubernetes
 - openshift
 - module

Reply:
Addressed.